jax-inference-offloading/jax_inference_offloading/api/controller.proto

```diff
@@ -122,6 +122,11 @@ message ShutdownRequest {
 }
 message ShutdownResponse {}
 
+message StartCUDAProfilerRequest {}
+message StartCUDAProfilerResponse {}
+message StopCUDAProfilerRequest {}
+message StopCUDAProfilerResponse {}
+
 service CouplingController {
```

Comment:
When you have time, please start adding some documentation to these RPCs.

Reply:
Will do!